### PR TITLE
Updating Dockerfile example to use Fabric8 Docker Maven plug-in

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Not sure whether a JIRA issue is needed or not. Wanted to create one, but JIRA is inaccessible right now.

**Comment by Ladicek** [comment]
ok to test

File: docker/docker-jaxrs-dockerfile/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM openjdk:jre-alpine
```
Comment (Ladicek) [inline docker/docker-jaxrs-dockerfile/Dockerfile]: Perhaps use the same base image as the `docker/docker-jaxrs` example? Not sure which one is "more official" :-) It probably doesn't really matter.

Comment (gunnarmorling) [reply docker/docker-jaxrs-dockerfile/Dockerfile]: Note that the Dockerfile existed before, I just had to move it, but didn't alter the base image.

Comment (Ladicek) [reply docker/docker-jaxrs-dockerfile/Dockerfile]: Ah, didn't notice that, as Git[Hub] didn't recognize it's a file move :-)

Comment (gunnarmorling) [reply docker/docker-jaxrs-dockerfile/Dockerfile]: Yes, I think that's because I had to add one argument to the entry point (it failed without that).

**Pull request merged**